distribution/helm/vector-agent/templates/daemonset.yaml

```diff
@@ -61,12 +61,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-            {{- if or .Values.hostMetricsSource.enabled (not .Values.customConfig) }}
             - name: PROCFS_ROOT
```

Comment:
How would someone disable this mounting if they wanted to?

Reply:
They can't with the change - though this does track with the mounting for kubernetes logs, which can't be turned off either

Reply:
👍 maybe this is something we could improve in the future.